.github/workflows/kernel_tests.yaml

```diff
@@ -0,0 +1,77 @@
+name: Test drivers against a matrix of kernels/distros
+
+on:
+  workflow_dispatch:
+    inputs:
+      libsversion:
+        description: libs version to be tested
+        type: string
+        required: false
+        default: master
+  push:
+    branches:
+      - master
+    
+concurrency:
+  group: kernel-tests-master
+  cancel-in-progress: true
+
+jobs:
+  test-kernels:
+    strategy:
+      fail-fast: false
+      matrix:
+        architecture: [X64, ARM64]
+    runs-on: [ "self-hosted", "linux", "${{matrix.architecture}}" ]    
```

Comment:
This PR is wip because we still need to port self-hosted runner from `alacuku/e2e-falco-tests` repo (https://github.com/alacuku/e2e-falco-tests) to falcosecurity/libs.